libs/application/ui-fields/src/lib/CheckboxFormField/CheckboxFormField.tsx

```diff
@@ -69,6 +74,7 @@ export const CheckboxFormField: FC<Props> = ({
           onSelect={onSelect}
           split={width === 'half' ? '1/2' : '1/1'}
           backgroundColor={backgroundColor}
+          rightContent={rightContent}
```

Comment:
Was this change to the application system approved by design and added to the design system?
Also if this is a change to the field you should update the checkboxformfield story in storybook to demo the changes.